packages/ember-engines/tests/acceptance/lazy-routeable-engine-test.js

```diff
@@ -81,15 +81,15 @@ module('Acceptance | lazy routable engine', function (hooks) {
     );
   }
 
-  test('it should pause to load JS and CSS assets on deep link into a lazy Engine', async function (assert) {
+  test('@classiconly it should pause to load JS and CSS assets on deep link into a lazy Engine', async function (assert) {
```

Comment:
@void-mAlex should you add `@classiconly` tag to the module (now it says `Acceptance | lazy routable engine`) so you don't add it to each individual test?

Reply:
there are three tests in the module that don't have the identifier